lib/generators/suspenders/styles_generator.rb

```diff
@@ -0,0 +1,52 @@
+module Suspenders
+  module Generators
+    class StylesGenerator < Rails::Generators::Base
+      include Suspenders::Generators::APIAppUnsupported
+
+      CSS_OPTIONS = %w[tailwind sass].freeze
+
+      class_option :css, enum: CSS_OPTIONS
+
+      def add_gems
+        return if css.nil?
+
+        case css
+        when "tailwind"
+          gem "tailwindcss-rails"
+        when "sass"
+          gem "dartsass-rails"
+        end
+
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def run_installation_scripts
+        return if css.nil?
+
+        case css
+        when "tailwind"
+          run "bin/rails tailwindcss:install"
+        when "sass"
+          run "bin/rails dartsass:install"
+        end
+      end
+
+      def build_directory_structure
+        empty_directory "app/assets/stylesheets/base"
+        create_file "app/assets/stylesheets/base/.gitkeep"
+
+        empty_directory "app/assets/stylesheets/components"
+        create_file "app/assets/stylesheets/components/.gitkeep"
+
+        empty_directory "app/assets/stylesheets/utilities"
+        create_file "app/assets/stylesheets/utilities/.gitkeep"
+      end
```

Comment:
Do we want to do this for applications running Tailwind?

Reply:
We'll keep these since Tailwind supports [@layer directives](https://tailwindcss.com/docs/functions-and-directives#layer).